.github/workflows/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug Report
+about: To report a bug
+title: ""
+labels: 'bug'
+assignees: ''
+
+---
+
+## **Bug report**
+
+**Describe the bug**
+Provide a clear and concise description of what the bug is. 
+
+**Steps to reproduce the bug** 
+Steps to reproduce the behaviour:
+1. Go to '...'
+2. Tap on '....'
+3. Scroll down to '....'
+4. See '....'
+
+**Expected behaviour**
+Provide a clear and concise description of what you expected to happen.
+
+**Observed behaviour**
+Provide a clear and concise description of what actually happened.
+
+**Screenshots:** 
+Visual examples can help us better understand the issue.
+
+**Platform:** 
```

Comment:
The convention would be to talk about the "Environment" when discussing the following matters. I have no particular problem with the use of the word "Platform", except that it may be confusing to some users. For this reason, I would advocate using the conventional name.